hack/mkdocs-generate-conformance.py

```diff
@@ -50,28 +50,49 @@ def on_pre_build(config, **kwargs):
                                                 'HTTPRoutePathRedirect', 'HTTPRouteHostRewrite', 'HTTPRoutePathRewrite', 'HTTPRouteRequestMirror', 'HTTPRouteRequestMultipleMirrors', 'HTTPRouteRequestTimeout', 'HTTPRouteBackendTimeout', 'HTTPRouteParentRefPort']
 
 
+# NOTE add past versions of implementations to here when new release is cut
+past_versions = """
+??? info
+
+
+    The latest release of Gateway API is shown here. Past Gateway API implementations and feature support can be found here:
+
+    - [v1.0.0](./implementation-table-v1.0.0.md)
+"""
+
+
 def generate_conformance_tables(reports):
 
-    gateway_http_table = generate_profiles_report(reports, 'HTTP')
+    gateway_http_table = generate_profiles_report(reports, 'GATEWAY-HTTP')
```

Comment:
I think this means that when we merge this, we'll be unable to update the v1.0 table with this script? Maybe we should keep an old version of the script around to keep the v1.0 table updated.

Reply:
gotcha - i think I'll add a check on the latest version to avoid creating another script 👍